gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/OrchestratorTest.java

```diff
@@ -370,16 +376,18 @@ public void onAddSpecForScheduledFlow() throws IOException {
     FlowSpec flowSpec = FlowSpec.builder().withConfig(config).build();
     AddSpecResponse response = new AddSpecResponse<>(new Object());
     Mockito.when(specCompiler.onAddSpec(flowSpec)).thenReturn(response);
-    AddSpecResponse addSpecResponse = dagMgrNotFlowLaunchHandlerBasedOrchestrator.onAddSpec(flowSpec);
+    AddSpecResponse addSpecResponse = orchestrator.onAddSpec(flowSpec);
     Assert.assertNotNull(addSpecResponse);
+    // Verifying that for scheduled flow isLeaseAcquirable is not called
+    Mockito.verify(dagManagementStateStore, Mockito.times(0)).existsCurrentlyLaunchingSimilarFlow(anyString(), anyString(), anyLong());
```

Comment:
comment names wrong method

also, use `Mockito.never()`

Reply:
Sure, Thanks for pointing out, updated